### PR TITLE
:memo: add note about future SDK team

### PR DESCRIPTION
**Pull request by JoshuaTheMiller**
### Why:

There has been some concern across communities regarding the Warning on the GitHub API Library page (the text being modified by this PR).

References (not closes): https://github.com/github/docs/issues/14926
References: https://github.com/octokit/octokit.js/discussions/620

### What's being changed:

This adds some detail and a link around the future of the SDK teams and a link to an official announcement.

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).

image

- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by JoshuaTheMiller** [comment]
My original wording is admittedly a bit on the cheesy side, but I wanted to get this ball rolling. Please don't hesitate to suggest alternatives if the basis of this change seems valid!

**Comment by ramyaparimi** [comment]
@JoshuaTheMiller 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/rest/overview/libraries.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ topics:
 
 {% warning %}
 
-Warning: As of late October 2021, the offical Octokit libraries are not currently maintained.
+Warning: As of late October 2021, the offical Octokit libraries are not currently maintained. However, please note that a dedicated SDK team is being built! For more information, reference [Issue #620 in the octokit.js repository](https://github.com/octokit/octokit.js/discussions/620).
```
Comment (skedwards88) [inline content/rest/overview/libraries.md:29]:
```suggestion
Warning: As of late October 2021, the offical Octokit libraries are not currently maintained. For more information, see [this discussion in the octokit.js repository](https://github.com/octokit/octokit.js/discussions/620).
```

**Review by skedwards88 — COMMENTED** [review]
Thanks for adding this additional reference! I'm going to modify the wording to fit the docs style, then we'll get this merged down for you.

**Review by skedwards88 — APPROVED** [review]

**Pull request merged**